### PR TITLE
[incubator/azuremonitor-containers] updates for agent release ciprod09162020 & win-ciprod09162020

### PR DESCRIPTION
**Pull request by ganga1980**
#### What this PR does / why we need it:
This PR has the changes related to adding azure monitor metrics support for azure arc k8s cluster.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/azuremonitor-containers/Chart.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ keywords:
   - kubernetes
   - kuberneteshealth
 home: https://docs.microsoft.com/en-us/azure/monitoring/monitoring-container-health
-icon: https://raw.githubusercontent.com/Microsoft/OMS-docker/ci_feature/img/azuremonitor-containers.svg
+icon: https://raw.githubusercontent.com/microsoft/Docker-Provider/ci_prod/img/azuremonitor-containers.svg
```
Comment (lachie83) [inline incubator/azuremonitor-containers/Chart.yaml]: I get a 404 for this URL

Comment (ganga1980) [reply incubator/azuremonitor-containers/Chart.yaml]: > I get a 404 for this URL

Thanks for pointing.  and just fixed.

**Pull request merged**